src/FluentValidation/ValidatorMetadata.cs

```diff
@@ -75,10 +75,10 @@ public class PropertyValidatorOptions {
 		public Func<PropertyValidatorContext, object> CustomStateProvider { get; set; }
 
 		/// <summary>
-		/// Severity of error.
+		/// Function used to retrieve the severity for the validator
 		/// </summary>
-		public Severity Severity { get; set; }
-		
+		public Func<PropertyValidatorContext, Severity> CustomSeverityProvider { get; set; }

```

Comment:
Let's rename to `SeverityProvider` as there's no longer a distinction between Severity / CustomSeverity.

Reply:
Done